README.md

```diff
@@ -89,6 +89,14 @@ You may also want to read this [introduction and sample use case][1].
 
 ## Installation
 
+### Arch AUR
+
+```bash
+sudo yay -S mkjson-git
```

Comment:
Could you remove the `sudo` part?


`yay` warns about running under root/sudo:

>  -> Avoid running yay as root/sudo.

Reply:
I deleted the forked repo, so I can't make additional changes to this PR... I think it is better to fix this yourself after merging this PR!